app/assets/javascripts/hubstats/application.js

```diff
@@ -14,45 +14,83 @@
 //= require jquery_ujs
 //= require select2
 //= require hubstats/bootstrap
+//= require bootstrap-datepicker
 //= require_tree .
 
 $(document).ready( function() {
+  defaultDateRange()
   setDateRange()
 });
 
+function defaultDateRange() {
```

Comment:
This function doesn't do anything with a date range so the name is a bit misleading.  Consider calling it something like `initDatePicker`.